src/python/tools/configuration/marius_config.py

```diff
@@ -480,6 +481,11 @@ def merge(self, input_config: DictConfig):
 
         if "dataset" in input_config.keys():
             self.dataset.merge(input_config.dataset)
+        
+        if "params_output_dir" in input_config.keys():
+            self.params_output_dir = input_config.params_output_dir
+        else:
+            self.params_output_dir = self.dataset.base_directory + "/model"
```

Comment:
If you have two config files which don't specify the model_dir (and thus default to <base_dir>/model), then they will overwrite each other. To avoid this we should add an incrementing integer to the end of the directory name. 

So if `<base_dir>/model_0` already exists and we want to train another model, this should be stored in the directory `<base_dir>/model_1` and so on.